reconcile/utils/terraform_client.py

```diff
@@ -575,34 +576,101 @@ def cleanup(self):
         for _, wd in self.working_dirs.items():
             shutil.rmtree(wd)
 
-    def _is_ignored_rds_modification(
+    def _can_skip_rds_modifications(
         self, account_name: str, resource_name: str, resource_change: Mapping[str, Any]
     ) -> bool:
+        """Skip pending RDS modifications.
+
+        Determine whether the RDS resource has pending modifications to the
```

Comment:
Thank you for the docstring! I'm going to approve the PR, but wanted to share our style guide on docstrings: https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-style-guide/#docstrings

I don't think we do a great job of making people aware that there's a style guide, so wouldn't certainly be open to your suggestions for making that better for new team members. Also, we could probably build more of these things into CI with automated checks. This one really isn't a major issue, but it's also in theory minimal effort to have our edits use the same docstring style.

Reply:
@steveteahan, updated to our preferred format.

Thank you!